Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1-alpine
+
+ENV PACKAGE_URL "https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Analytics-Middle-Tier"
+ENV PACKAGE_NAME "EdFi.AnalyticsMiddleTier"
+ENV AMT_VERSION "2.8.0"
```

Comment:
We probably want to change this a bit so that we can pass in the value as an argument. I will look for an example.

Reply:
Well, there is probably a better way, but this is 100% consistent with how the other Ed-Fi Docker containers are setup ([example](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Docker/blob/2a167c85b51fa132db37d08310e286ffd2b72337/Web-Ods-Api/Alpine/mssql/Dockerfile#L14)). So this is fine. Someone just has to remember to update this hard-coded version number whenever there is a new release.